src/avram/model.cr

```diff
@@ -168,7 +172,11 @@ class Avram::Model
   macro setup_getters(columns, *args, **named_args)
     {% for column in columns %}
       def {{column[:name]}}
-        {{ column[:type] }}::Lucky.from_db! @{{column[:name]}}
+        {% if column[:type].is_a?(Generic) %}
+          {{column[:type].type_vars.first}}::Lucky.from_db! @{{column[:name]}}, array: true
```

Comment:
This would mean every `from_db!` method would now need an array option... This becomes very messy 😢